core-examples/node-js/abi.encode_boc/index.js

```diff
@@ -0,0 +1,30 @@
+const { TonClient } = require("@tonclient/core");
+const { libNode } = require("@tonclient/lib-node");
+
+TonClient.useBinaryLibrary(libNode);
+
+(async () => {
+    const client = new TonClient({});
+    try {
+        // Encoding ABI parameters values into a BOC:
+        const boc = (await client.abi.encode_boc({
+            params: [
+                { name: "dest", type: "address" },
+                { name: "value", type: "uint128" },
+                { name: "bounce", type: "bool" },
+            ],
+            data: {
+                "dest": "-1:3333333333333333333333333333333333333333333333333333333333333333",
+                "value": 1234567,
```

Comment:
Maybe here is  a problem.... 
If some one tries to pass a large number here, they will get an error. 
For example value: 1000000000000000000000,
```
  message: 'Invalid ABI specified: Wrong data format:\n1e21',
```
Meanwhile this will work:
 value: "1000000000000000000000"
 OR
  value: BigInt(1000000000000000000000)

Might it be better always pass numbers as strings  or BigNumber ???